etc/avocado/conf.d/vt_joblock.conf

```diff
@@ -0,0 +1,4 @@
+[plugins.vtjoblock]
```

Comment:
Also as mentioned in https://github.com/avocado-framework/avocado-vt/pull/484/commits/d8b43798ffe8b568878efcc8452180d23b72897d would you please add this file to `make link` target?